proto/src/code_execution.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
```

Comment:
Check naming and types. We use i32 for IDs in server repo so assuming we want to use it throughout

Reply:
Seems better to use uint32?

Reply:
Agree. In some sense we probably want to be consistent across the codebase though since otherwise we'll have to cast the id in the request/response every time

Generally I think using uint32 should be the default for our IDs

Reply:
> In some sense we probably want to be consistent across the codebase though since otherwise we'll have to cast the id in the request/response every time.

Yeah I was suggesting we use u32 everywhere.

But looked into this a bit – Postgres doesn't support uint natively so we'd need to use `bigint` (64 bit integer). We can pretty easily make the switch down the line (both proto definition and DB migration) so let's stick with i32 for now.